pkg/gardenlet/controller/federatedseed/seed_control.go

```diff
@@ -166,15 +162,15 @@ func (c *Controller) createReconcileSeedRequestFunc(ctx context.Context) reconci
 	return func(req reconcile.Request) (reconcile.Result, error) {
 		seed, err := c.seedLister.Get(req.Name)
 		if apierrors.IsNotFound(err) {
-			logger.Logger.Infof("Skipping - Seed %s was not found", req.Name)
+			logger.Logger.Infof("Skipping federated seed setup - Seed %s was not found", req.Name)
```

Comment:
Can you also switch the seed informer and lister in the top-level controller to the controller-runtime informer and client?

https://github.com/gardener/gardener/blob/6e528916105623fee733d2b3c043385375a08761/pkg/gardenlet/controller/federatedseed/seed_control.go#L84-L91
and
https://github.com/gardener/gardener/blob/6e528916105623fee733d2b3c043385375a08761/pkg/gardenlet/controller/federatedseed/seed_control.go#L163